### PR TITLE
feat: set the correct mode when loading the page

### PR DESCRIPTION
**Pull request by zqran**
When the page is converted to `edgeless` mode and reloaded, the editor should be in `edgeless` mode

### CODE REVIEW

**Review by Himself65 — APPROVED** [review]

**Comment by zqran** [comment]
When the page is in `edgeless` mode, there is no Title, which causes the test to fail.

Should it be handled differently depending on the page mode?

**Comment by darkskygit** [comment]
Hello! Thank you for your contribution.

In order for us to review and merge your suggested changes, please sign at https://github.com/toeverything/AFFiNE/edit/master/.github/CLA.md

**Comment by darkskygit** [comment]
> When the page is in `edgeless` mode, there is no Title, which causes the test to fail.
> 
> Should it be handled differently depending on the page mode?

you can adjust the test cases. judging from the current design, there is indeed no title in edgeless.

**Review by Himself65 — CHANGES_REQUESTED** [review]
Check your git history

**Comment by zqran** [comment]
> Check your git history

yeah, i'm checking~

File: tests/quick-search.spec.ts [hunk 1]
```diff
@@ -9,18 +9,24 @@ const openQuickSearchByShortcut = async (page: Page) =>
   await withCtrlOrMeta(page, () => page.keyboard.press('k', { delay: 50 }));
 
 async function assertTitle(page: Page, text: string) {
-  const locator = page.locator('.affine-default-page-block-title').nth(0);
-  const actual = await locator.inputValue();
-  expect(actual).toBe(text);
+  const edgeless = page.locator('affine-edgeless-page');
+  if (!edgeless) {
```
Comment (zqran) [inline tests/quick-search.spec.ts:13]:
```suggestion
  const edgeless = page.locator('affine-edgeless-page');
  if (!edgeless) {
```

I'm not very familiar with the testing of this project, will this work?

Comment (himself65) [reply tests/quick-search.spec.ts:14]: no

Comment (himself65) [reply tests/quick-search.spec.ts:14]: you need `await page.locator('affine-edgeless-page').toHaveCount(0);`

**Review by Himself65 — APPROVED** [review]

**Pull request merged**